modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v1/structure.rb

```diff
@@ -17,10 +17,10 @@ def form_fill
             form_fields.veteran_ssn => form_data.veteran_ssn,
             form_fields.veteran_file_number => form_data.veteran_file_number,
             form_fields.veteran_dob => form_data.veteran_dob,
-            form_fields.mailing_address_number_and_street => form_data.mailing_address_number_and_street,
+            form_fields.mailing_address => form_data.mailing_address,
```

Comment:
How large can this get before the pdf field fails? Do you think it'd make sense to turn this into an autosize box (as a separate ticket)?

Reply:
I was going to, but then tested it out and it fits quite a bit. I'll test again and let you know exactly how much.

Reply:
Mailing address can take in 165 characters the way it is now. Thoughts?

Reply:
The combining of the...5+ fields that make up the address likely mean we'll need to create a custom validation so that the combined mailing list doesn't exceed 165 characters. We should also add a `description` to the address schema noting that the maximum combined character limit is 165.